Source/ChocolateyGui/Views/SettingsView.xaml

```diff
@@ -75,14 +75,15 @@
                             <RowDefinition Height="Auto"/>
                             <RowDefinition Height="*"/>
                         </Grid.RowDefinitions>
-                        <TextBlock Margin="5 5 5 3" Style="{StaticResource SubheaderTextBlockStyle}" Text="{x:Static lang:Resources.SettingsView_ChocoGui}"/>
+                        <TextBlock Margin="5 5 5 3" Style="{StaticResource SubheaderTextBlockStyle}" Text="{lang:Localize SettingsView_ChocoGui}" />
                         <ScrollViewer Margin="5 0" Grid.Row="1" Width="300" HorizontalAlignment="Left" VerticalScrollBarVisibility="Auto">
                             <ItemsControl ItemTemplate="{StaticResource ChocolateyGUISettingTemplate}">
                                 <ItemsControl.Items>
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_ToggleShowConsoleOutputTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleShowConsoleOutputDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.ShowConsoleOutput}" />
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewLocalTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewLocalDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.DefaultToTileViewForLocalSource}" />
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewRemoteTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewRemoteDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.DefaultToTileViewForRemoteSource}" />
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_UseDelayedSearchTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleUseDelayedSearchDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.UseDelayedSearch}" />
+                                    <metro:SplitButton ItemsSource="{Binding Source={StaticResource BindingProxy}, Path=Data.AllLanguages}" SelectedItem="{Binding Source={StaticResource BindingProxy}, Path=Data.UseLanguage}" />

```

Comment:
Why using a SplitButton?

Reply:
No special reason, just something that popped into mind since I couldn't find a `DropDownList`.

Reply:
The ComboBox has a DropDownList, or did you have another requirement?

Reply:
Oh right, completely forgot about ComboBox. Been a while since I last worked in WPF.